server.py

```diff
@@ -60,6 +61,17 @@ def auth_path_prefix():
 @app.before_request
 @optional_auth
 def assert_user_is_logged():
+    public_endpoints = ['healthz', 'ready']
+    if request.endpoint in public_endpoints:
+        return
+
+    tenant = tenant_handler.tenant()
+    config_handler = RuntimeConfig("mapViewer", app.logger)
+    config = config_handler.tenant_config(tenant)
+    public_paths = config.get("public_paths", [])
+    if request.path in public_paths:
+        return
+
```

Comment:
@smaspons @manisandro - as far as I understand, the code adds two new URLs, `.../healthz` and `.../ready` that get served by the service without authentication.

That means that the map-viewer becomes discoverable without authentication, which is an information leak security-wise.
 
Additionaly it's now possible to add other patchs that would get the same treatment.

Wouldn't it be better to make `.../healthz` and `.../ready` optional as well and let the user add them to the `public_paths` if she chooses to do so?